api/pkg/transformer/pipeline/environment.go

```diff
@@ -28,6 +28,11 @@ func NewEnvironment(compiledPipeline *CompiledPipeline, logger *zap.Logger) *Env
 		logger:           logger,
 	}
 
+	// attach pre-loaded tables to environment
+	for k, v := range compiledPipeline.preloadedTables {
```

Comment:
there is a chance that the preloadtables has the same name with the other tables, how do we handle this?

Reply:
It should throw the same error as the normal way to create tables?

Reply:
Shall we create warning logs if there is duplicated name?

Reply:
Shall we create warning logs if there is duplicated name?

Reply:
i thought `preloadedTables` is table that created only from file, not the table that use `fromFile` `baseTable`. With current approach if there are two table creation that use same file as baseFile it will download and parse multiple time ?